typesense-minibar.js

```diff
@@ -146,7 +149,7 @@ globalThis.tsminibar = function tsminibar (form) {
     listbox.hidden = !state.open;
     form.classList.toggle('tsmb-form--open', state.open);
     if (state.open) {
-      listbox.innerHTML = (state.hits.map((hit, i) => `<div role="option"${i === state.cursor ? ' aria-selected="true"' : ''}>${hit.lvl0 ? `<div class="tsmb-suggestion_group">${hit.lvl0}</div>` : ''}<a href="${hit.url}" tabindex="-1"><div class="tsmb-suggestion_title">${hit.title}</div><div class="tsmb-suggestion_content">${hit.content}</div></a></div>`).join('') || `<div class="tsmb-empty">No results for '${escape(state.query)}'.</div>`) + (form.dataset.foot ? '<a href="https://typesense.org" class="tsmb-foot" title="Search by Typesense"></a>' : '');
```

Comment:
`String#replaceAll` is slightly too new. It cuts off a few too many Chrome versions, I think. Compare <https://github.com/jquery/typesense-minibar#browser-support> to <https://caniuse.com/?search=replaceAll>.

I suggest using `.replace()` instead. If we want to support multiple substitutions for the same string, we could a regex like `.replace(/%s/g, query)` instead!

Reply:
Good catch! I've just replaced `replaceAll` with plain `replace`. I don't think that in reality anybody will actually include the queried text twice in the message, so let's keep it simple.